packages/lit-dev-discord-bot/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "lib",
+    "incremental": true,
+    "tsBuildInfoFile": "lib/.tsbuildinfo"
```

Comment:
We usually put the build info files next to tsconfigs as `tsconfig.tsbuildinfo`, then gitignore them

Reply:
That seems to be how someone set up the entire repo 🤷